maintenance/static/css/main.css

```diff
@@ -7377,47 +7377,46 @@ a.text-gray-dark:focus, a.text-gray-dark:hover {
 
 /* keep this #f4f4f4 as this is the background colour that product images have */
-/* mavis */
-.donate-header::before {
+.heart-glyph::before {
```

Comment:
I basically just copied and pasted content from main site's `main.compiled.css` + updated a few file paths so things show up correctly on maintenance page.